### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
# What

- Cleaner floating logo
- Use uniform badges
- Use text from Misskey-hub

# Why

To keep readme more in line with Misskey-hub

# Additional info
[Preview](https://github.com/ThatOneCalculator/misskey/blob/readme/README.md)

### CODE REVIEW

**Comment by syuilo** [comment]
- The animated logo has its pros and cons, but I thought it was funny.
- The badges were colorful, eye-catching, and I love them.

File: README.md [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
-•
-[📦 Create your own instance](https://misskey-hub.net/docs/install.html)
-•
-[🛠️ Contribute](./CONTRIBUTING.md)
-•
-[🚀 Join the community](https://discord.gg/Wp8gVStHW3)
-
+	
+<a href="https://misskey-hub.net/instances.html">
+		<img src="https://custom-icon-badges.herokuapp.com/badge/find_an-instance-acea31?logoColor=acea31&style=for-the-badge&logo=misskey&labelColor=363B40" /></a>
```
Comment (Johann150) [inline README.md]:
```suggestion
		<img src="https://custom-icon-badges.herokuapp.com/badge/find_an-instance-acea31?logoColor=acea31&style=for-the-badge&logo=misskey&labelColor=363B40" alt="find an instance"/></a>	
```

File: README.md [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
-[🛠️ Contribute](./CONTRIBUTING.md)
-•
-[🚀 Join the community](https://discord.gg/Wp8gVStHW3)
-
+	
+<a href="https://misskey-hub.net/instances.html">
+		<img src="https://custom-icon-badges.herokuapp.com/badge/find_an-instance-acea31?logoColor=acea31&style=for-the-badge&logo=misskey&labelColor=363B40" /></a>	
+	
+<a href="https://misskey-hub.net/docs/install.html">
+		<img src="https://custom-icon-badges.herokuapp.com/badge/create_an-instance-FBD53C?logoColor=FBD53C&style=for-the-badge&logo=server&labelColor=363B40" /></a>
```
Comment (Johann150) [inline README.md]:
```suggestion
		<img src="https://custom-icon-badges.herokuapp.com/badge/create_an-instance-FBD53C?logoColor=FBD53C&style=for-the-badge&logo=server&labelColor=363B40" alt="create an instance"/></a>	
```

File: README.md [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
-
+	
+<a href="https://misskey-hub.net/instances.html">
+		<img src="https://custom-icon-badges.herokuapp.com/badge/find_an-instance-acea31?logoColor=acea31&style=for-the-badge&logo=misskey&labelColor=363B40" /></a>	
+	
+<a href="https://misskey-hub.net/docs/install.html">
+		<img src="https://custom-icon-badges.herokuapp.com/badge/create_an-instance-FBD53C?logoColor=FBD53C&style=for-the-badge&logo=server&labelColor=363B40" /></a>	
+
+<a href="./CONTRIBUTING.md">
+		<img src="https://custom-icon-badges.herokuapp.com/badge/become_a-contributor-A371F7?logoColor=A371F7&style=for-the-badge&logo=git-merge&labelColor=363B40"/>
```
Comment (Johann150) [inline README.md]:
```suggestion
		<img src="https://custom-icon-badges.herokuapp.com/badge/become_a-contributor-A371F7?logoColor=A371F7&style=for-the-badge&logo=git-merge&labelColor=363B40" alt="become a contributor"/>
```

File: README.md [hunk 4]
```diff
@@ -1,27 +1,32 @@
...
+</a>
+
+<a href="https://discord.gg/Wp8gVStHW3">
+		<img src="https://custom-icon-badges.herokuapp.com/badge/join_the-community-5865F2?logoColor=5865F2&style=for-the-badge&logo=discord&labelColor=363B40"/>
```
Comment (Johann150) [inline README.md]:
```suggestion
		<img src="https://custom-icon-badges.herokuapp.com/badge/join_the-community-5865F2?logoColor=5865F2&style=for-the-badge&logo=discord&labelColor=363B40" alt="join the community"/>
```

File: README.md [hunk 5]
```diff
@@ -1,27 +1,32 @@
...
+</a>
+	
+<a href="https://www.patreon.com/syuilo">
+		<img src="https://custom-icon-badges.herokuapp.com/badge/become_a-patron-F96854?logoColor=F96854&style=for-the-badge&logo=patreon&labelColor=363B40" />
```
Comment (Johann150) [inline README.md]:
```suggestion
		<img src="https://custom-icon-badges.herokuapp.com/badge/become_a-patron-F96854?logoColor=F96854&style=for-the-badge&logo=patreon&labelColor=363B40" alt="become a patron"/>
```

File: README.md [hunk 6]
```diff
@@ -30,17 +35,16 @@
 
 ## ✨ Features
 - **ActivityPub support**\
-	It is possible to interact with other software.
+Not on Misskey? No problem! Not only can Misskey instances talk each other, but you can make friends with people on other networks like Mastodon and Pixelfed!
```
Comment (Johann150) [inline README.md]:
```suggestion
Not on Misskey? No problem! Not only can Misskey instances talk to each other, but you can make friends with people on other networks like Mastodon and Pixelfed!
```

File: README.md [hunk 7]
```diff
@@ -30,17 +35,16 @@
...
 - **Reactions**\
-	You can add "reactions" to each post, making it easy for you to express your feelings.
+You can add emoji reactions to any post! No longer are you bound by a like button, show everyone exactly how you feel with the tap of a button.
 - **Drive**\
-	An interface to manage uploaded files such as images, videos, sounds, etc.
-	You can also organize your favorite content into folders, making it easy to share again.
+Hate not being able to find something you uploaded? With Drive, you get managed and powerful cloud storage right in your social media!
```
Comment (Johann150) [inline README.md]: In my opinion these previous parts were fine, and the new phrasing sounds too enthusiastic.

File: README.md [hunk 8]
```diff
@@ -30,17 +35,16 @@
...
 - **Rich Web UI**\
-	Misskey has a rich WebUI by default.
-	It is highly customizable by flexibly changing the layout and installing various widgets and themes.
-	Furthermore, plug-ins can be created using AiScript, a original programming language.
-- and more...
+	Misskey has a rich and complex (but easy to use) Web UI!
```
Comment (Johann150) [inline README.md]: "complex but easy to use" sounds very much like a contradiction. The adjective "rich" already expresses that idea so I don't think we need that strange wording.

```suggestion
	Misskey has a rich web user interface!
```

I think we could even say that Misskey has multiple web user interfaces, too.

File: README.md [hunk 9]
```diff
@@ -30,17 +35,16 @@
...
+	It's highly customizable, from changing the layout, adding widgets and making themes.
```
Comment (Johann150) [inline README.md]: Better not use an apostrophe in a more formal text like this. The "from" doesn't really make sense this way, maybe something like:
```suggestion
	It is highly customizable, from changing the layout and adding widgets to making custom themes.
```

File: README.md [hunk 10]
```diff
@@ -30,17 +35,16 @@
...
+	Furthermore, plugins can be created using AiScript, an original programming language.
+- And so more...
```
Comment (Johann150) [inline README.md]:
```suggestion
- And much more...
```

**Review by Johann150 — CHANGES_REQUESTED** [review]
You added alt text to the first image, but then didn't add alt text to the other ones.

I'm also not totally convinced of the phrasing. Please note: I did not review <https://github.com/misskey-dev/misskey-hub/pull/81> so the same might apply to there.

File: README.md (hunk 10 above)
Comment (ThatOneCalculator) [reply README.md]: Yep, typo on my part

File: README.md (hunk 7 above)
Comment (ThatOneCalculator) [reply README.md]: It's still the phrasing used on the website

File: README.md (hunk 8 above)
Comment (ThatOneCalculator) [reply README.md]: How about "Misskey has a rich, customizable, and easy to use web user interface!"

Comment (Johann150) [reply README.md]: It already says that it is customizable on the next line down. But I could go with easy to use.

Comment (ThatOneCalculator) [reply README.md]: How about
```
	Misskey has a rich and easy to use Web UI!
	It is highly customizable, from changing the layout and adding widgets to making custom themes.
	Furthermore, plugins can be created using AiScript, an original programming language.
```

File: README.md (hunk 7 above)
Comment (Johann150) [reply README.md]: That might be the case, but that does not change my opinion of the text. See the note above:
> Please note: I did not review https://github.com/misskey-dev/misskey-hub/pull/81 so the same might apply to there.

Comment (ThatOneCalculator) [reply README.md]: @Johann150 What do you think would be better phrasing for this section?

Comment (Johann150) [reply README.md]: > In my opinion these previous parts were fine

**Review by Johann150 — APPROVED** [review]

**Comment by ThatOneCalculator** [comment]
@syuilo seems everything's good now :)

**Comment by syuilo** [comment]
👍👍👍

**Pull request merged**